Gems/ROS2/Code/Source/Lidar/ROS2LidarSensorComponent.h

```diff
@@ -47,6 +48,7 @@ namespace ROS2
 
         bool m_canRaycasterPublish = false;
         std::shared_ptr<rclcpp::Publisher<sensor_msgs::msg::PointCloud2>> m_pointCloudPublisher;
+        std::shared_ptr<rclcpp::Publisher<vision_msgs::msg::LabelInfo> > m_segmentationClassesPublisher;
```

Comment:
Was this verified by a person or team that is more familiar than me with processing such kind of data?

Reply:
I think we established that this is the right kind of message for this with Adam